catalog-info.yaml

```diff
@@ -0,0 +1,57 @@
+# Declare a Backstage Component that represents your application.
+---
+# yaml-language-server: $schema=https://json.schemastore.org/catalog-info.json
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: go-structform
+
+spec:
+  type: library
+  owner: group:ingest-fp
+  system: platform-ingest
+  lifecycle: production
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
```

Comment:
I actually had a discussion with the ci-systems team about this (because it's pinned to a specific version and doesn't include the latest information for RRE) and we should instead be using:

https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json

we should also raise PRs to fix old occurrences.

internal reference https://elasticco.atlassian.net/browse/CI-322?focusedCommentId=96184